lms/static/certificates/sass/_print.scss

```diff
@@ -94,6 +94,12 @@
 // ------------------------------
 @media print {
 
+    html,
+    body {
+        height: 650px;
```

Comment:
Hmm, I really don't like this px value, but can't find another alternative after researching a bit to force contents of the DOM only on one page. Can you do the following:

* check in different browsers and print settings (mostly landscape vs. portrait and 8.5x11 vs. A4) to see how this height affects end print rendering?
* create a Sass config-level variable for this value and plumb it through here, so that others when they customize their designs/renderings can override this?